pkg/dao/dao.go

```diff
@@ -27,7 +23,7 @@ type Dao struct {
 	config    Config
```

Comment:
I disagree slightly, the broker looks at the errors that are returned from the DAO as it could be etcd errors. 

We could get around that by adding another layer of predefined errors in the dao and in the broker.